src/Bindings/ManualBindings.h

```diff
@@ -427,14 +446,12 @@ class cManualBindings
 			virtual bool Item(Ty2 * a_Item) override
 			{
 				bool res = false;
-				if (!m_LuaState.Call(m_FnRef, a_Item, cLuaState::Return, res))
+				if (m_LuaState.Call(m_FnRef, a_Item, cLuaState::Return, res))
 				{
-					LOGWARNING("Failed to call Lua callback");
-					m_LuaState.LogStackTrace();
```

Comment:
I believe the failure in the callback is already reported to the console, including the stack trace, so there should be no need to report the failure again.